javascript/UserForm.js

```diff
@@ -721,6 +721,9 @@ jQuery(function ($) {
 		setInterval(function () {
 			$.ajax({ url: 'UserDefinedForm_Controller/ping' });
 		}, 180 * 1000);
+
+		// Bind a confirmation message when navigating away from a partially completed form.
+		$('form.userform').areYouSure();
```

Comment:
I'll get you to add the master string to the en.js file too, please, if that's ok. :) there's a javascript/lang/src/en.js and a javascript/lang/en.js file to update. If you update just the master (en.js) that's sufficient.